src/cf-tables/src/cf-tables.less

```diff
@@ -14,6 +14,9 @@
 @table-cell-bg:              #ffffff; // $color-white
 @table-cell-bg_alt:          #f1f1f1; // $color-gray-lightest
 @table-scrolling-border:     #e4e2e0; // $color-gray-warm-light
+@table-row-link-color:       #ffffff; // $color-white
```

Comment:
Fixed in https://github.com/cfpb/capital-framework/pull/396/commits/28ae6165675e1397f9cb358194d0277f9d9c38e5